src/components/Multisig.tsx

```diff
@@ -762,12 +764,498 @@ function AddTransactionDialog({
+            [uxpMint.toBuffer()],
+            UXDIDOProgramAdress
+        );
+        const poolSigner = _poolSigner;
+    // @ts-ignore
+        enqueueSnackbar("Creating USDC pool withdraw transaction", {
+            variant: "info",
+        });
+
+        const data = withdrawIdoUsdcPoolData(uxdClient);
+
+        const accounts = [
+            // HERE NEED TO ADD THE RIGHT ACCOUNTS -- Not sure what can be hardcoded or not your call for now.
+            // Accounts expected can be found here https://github.com/UXDProtocol/uxd_ido/blob/main/programs/uxd_ido/src/lib.rs#L281
+
+            // pool_account -- can be created arbitrarily - then will need to be referenced in the other operations and used on the front end (THE POINTER TO OUR IDO)
+            {
+                pubkey: new PublicKey(poolAccountAddr),
+                isWritable: true,
+                isSigner: true,
+            },
+            // pool_signer -- this is the multisig PDA : 35F3GaWyShU5N5ygYAFWDw6bGVNHnAHSe8RKzqRD2RkT
+            //? While testing we used to use a derivation from the uxp mint for creating this account
+            {
+                pubkey: poolSigner,
+                isWritable: false,
+                isSigner: true,
+            },
+
+            // pool_usdc --
+            {
+                pubkey: new PublicKey(poolUsdcAddr),
+                isWritable: false,
+                isSigner: false,
+            },
+            // distribution_authority -- should be the multisig PDA - might have an issue with the program cause it"s also a payer
+            // 35F3GaWyShU5N5ygYAFWDw6bGVNHnAHSe8RKzqRD2RkT
+            // NGMI if we discover issues last minute, this need to be done asap for our mental wellbeing, sorry to be lame but it"s super important
+            {
+                pubkey: multisigPDA,
+                isWritable: true,
+                isSigner: true,
+            },
+            // creator_usdc -- is actually the token account that will receive the usdc
+            {
+                pubkey: multisigUsdc,
+                isWritable: true,
+                isSigner: false,
+            },
+            // token_program
+            {
+                pubkey: TOKEN_PROGRAM_ID,
+                isWritable: false,
+                isSigner: false,
+            },
+
+            // clock
+            {
+                pubkey: anchor.web3.SYSVAR_CLOCK_PUBKEY,
+                isWritable: false,
+                isSigner: false,
+            },
+        ];
+        const transaction = new Keypair();
+        const txSize = 1000; // todo
+        itxs.push((await multisigClient.account.transaction.createInstruction(
+            transaction,
+            // @ts-ignore
+            txSize
+        )));
+        const tx = await multisigClient.rpc.createTransaction(
+            multisigClient.programId,
+            accounts,
+            data,
+            {
```

Comment:
^ So you call the multisig program `createTransaction`, sending the `multisigProgramID`, the `accounts` this specific transaction expect, and the `data` serialized as expected by this transaction.


Below you should be doing the same but for the "nested" instruction.

as said on slack earlier. You pass the multisig programID but that should be the `IDOprogramId`. The account should be the account the Initialize account expect. And you should probably by passing some data too. 

Remember to zoom out, and this "what is this instruction actually doing"

I'm not sure about where the data should be cause I don't have the IDE open, but please read the function signature of `multisig. createTransaction` and determine wether it should be passed where it is now or in the nesting

Reply:
also should have an result checking for the success to be dsplayed. non essential for now

Reply:
That one was quite the trick. See code update for how it is supposed to work

Reply:
yeah will do later, I'm planning maybe a small refactor tonight for more clarity and separating components into their own file, easier to navigate